nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -211,13 +209,29 @@ require(
 
         Reveal.addEventListener('slidechanged', update);
 
-        var update_scroll = function(event){
-          $(".reveal").scrollTop(0);
-        };
-
-        Reveal.addEventListener('slidechanged', update_scroll);
+        function setScrollingSlide(scroll) {
+
+            if (scroll === "True") {
+              var h = $('.reveal').height() * 0.95;
+              var hpx = "" + h + "px";
+              $('section.present').find('section')
+                .filter(function() {
+                  return $(this).height() > h;
+                })
+                .css('height', hpx)
+                .css('overflow-y', 'scroll')
+                .css('margin-top', '20px');
+            }
+        }
+
+        Reveal.addEventListener( 'slidechanged', function( event ) {
+          // check and set the scrolling slide every time the slide change
+          var scroll = "{{resources.reveal.scroll}}";
```

Comment:
I change my mind and I went with the json_dumps approach (and simplify the structure a little bit).